src/utils.cpp

```diff
@@ -417,6 +417,15 @@ std::string utils::convert_text(const std::string& text,
 	return result;
 }
 
+std::string utils::utf8_to_locale(const std::string& text)
+{
+	if (text.empty()) {
+		return {};
```

Comment:
Out of curiosity: Is this check required for correct functionality or is it used to optimize out unnecessary `convert_text()` calls?

Reply:
Original `RssItem::title()` had it, and it made sense as an optimization, so I copied it here. It was added [a long, long time ago](https://github.com/newsboat/newsboat/commit/ff4251954298c153bba0017e04e765ef179351fe#diff-8de95749c592883cd4aac68ec7281e57R360) without any clear explanation, so I have no idea about its intended purpose.